src/Illuminate/Auth/Access/Gate.php

```diff
@@ -392,13 +392,20 @@ public function authorize($ability, $arguments = [])
     public function inspect($ability, $arguments = [])
     {
         try {
-            $result = $this->raw($ability, $arguments);
+            $abilities = explode('|', $ability);
+            foreach ($abilities as $ability) {
+                $result = $this->raw($ability, $arguments);
 
-            if ($result instanceof Response) {
-                return $result;
+                if ($result instanceof Response) {
+                    return $result;
```

Comment:
Hi there! I am not really sure why have you expanded this check out? Was it not simpler and readable with the ternary operator? I think this was fine how it was to be honest.